src/devices/RelayBoard/RelayBoard.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+
+using RelayBoard;
+
+namespace Iot.Device.RelayBoard
```

Comment:
I'm having a hard time figuring out what's the benefit of RelayBoard class (based on GpioController) - if this was an I2C/SPI device with multiple relays I can see the interface being somewhat useful but in case of GpioController why not just use array directly? The reason I bring this up is we go ahead with this class there is no reason why we shouldn't have: ButtonBoard/KeyBoard, LedBoard/LedStrip (or if you go dimension further LedMatrix), PwmBoard, AdcBoard, DHT11Board and many other single pin devices... Hope this is making sense why I'm pushing back on this API

Reply:
The reason for this is due to the ability to abstract away the construction of the relays; for each relay board they are going to be the same type (Normally Open or Normally Closed) and use the same GpioController. With the RelayBoard class, you only have to worry about setting that property once. 

I went back and forth a lot on whether or not this was a good enough reason to include RelayBoard, and in the end I felt like there was enough of a benefit to add this while still allowing users to create the Relay directly (and then they could keep their own list/reference to each one).